.github/workflows/linearizability-nightly.yaml

```diff
@@ -1,25 +1,31 @@
 name: Linearizability Nightly
 on:
-  # schedules always run against the main branch
+  # schedules always run against the main branch, hence we have to create separate jobs
+  # with individual checkout actions for each of the active release branches
   schedule:
-    - cron: '25 9 * * *'
+    - cron: '25 9 * * *' # runs every day at 09:25 UTC
 jobs:
-  test:
-    runs-on: ubuntu-latest
+  test-main:
     # GHA has a maximum amount of 6h execution time, we try to get done within 3h
     timeout-minutes: 180
-    steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-go@v2
-      with:
-        go-version: "1.19.3"
-    - run: |
-        make gofail-enable
-        make build
-        mkdir -p /tmp/linearizability
-        cat server/etcdserver/raft.fail.go
-        EXPECT_DEBUG=true GO_TEST_FLAGS='-v --count 500 --failfast --run TestLinearizability' RESULTS_DIR=/tmp/linearizability make test-linearizability
-    - uses: actions/upload-artifact@v2
-      if: always()
-      with:
-        path: /tmp/linearizability/*
+    uses: ./.github/workflows/linearizability-workflow.yaml
+    with:
+      ref: main
+      count: 500
+      timeoutDuration: 3h
+  test-3.5:
```

Comment:
Note, older releases don't have same gofailpoints. Tests will fail when trying to trigger a failpoint that is present on main which is not available on release-3.4 nor release-3.5.

We might need to skip some of scenarios that are not yet backward compatible.

To make new failpoints work with older releases we would need:
* merge https://github.com/etcd-io/gofail/pull/44#discussion_r1043883246 that will ensure that failpoints have same name
* implement checking which failpoints are available and triggering only those available.

Reply:
Second point was implemented in https://github.com/etcd-io/etcd/pull/15046